normits_demand/efs_constants.py

```diff
@@ -233,8 +233,8 @@
 MODEL_NAME = 'noham'
 
 # YEARS
-BASE_YEAR = 2018
-FUTURE_YEARS = [2033, 2040, 2050]
+BASE_YEAR = 2021
```

Comment:
Possibly should be reverted - forecast doesn't run from here anymore but run_ntem_forecast still does